packages/context-menu/src/vaadin-context-menu.js

```diff
@@ -308,12 +308,6 @@ class ContextMenu extends ControllerMixin(ElementMixin(ThemePropertyMixin(ItemsM
       /** @private */
       _context: Object,
 
-      /** @private */
-      _boundClose: Object,
-
-      /** @private */
-      _boundOpen: Object,
```

Comment:
There is no need to store bound methods in `properties` block as we don't do that elsewhere.